pkg/k8shandler/visualization.go

```diff
@@ -148,7 +163,21 @@ func (clusterRequest *ClusterLoggingRequest) removeKibana() (err error) {
 
 func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaDeployment() (err error) {
 
-	kibanaPodSpec := newKibanaPodSpec(clusterRequest.cluster, "kibana", "elasticsearch")
+	proxyConfig := &configv1.Proxy{}
```

Comment:
shouldn't we already have this proxy object before in the call stack? It seems like we can save a k8s call by passing that object through...

Reply:
> shouldn't we already have this proxy object before in the call stack? It seems like we can save a k8s call by passing that object through...

You mean we should share the proxyConfig object (and the trustedCABundleCM) between kibana and fluentd?  Is it possible to stash the object somewhere and refer them from kibana and fluentd?  If so, where is the good candidate to stash the objects?  For instance, ClusterLoggingRequest?

Reply:
i'm not sure we need to stash it, but can't we pass it along from the controller?